android/quest/src/main/java/org/smartregister/fhircore/quest/ui/usersetting/UserSettingScreen.kt

```diff
@@ -313,11 +361,17 @@ fun UserSettingScreen(
       ) {
         Spacer(modifier = Modifier.weight(1f))
 
+        val resId = R.drawable.ic_opensrplogo
         Image(
-          painterResource(R.drawable.ic_opensrplogo),
+          painterResource(resId),
```

Comment:
```suggestion
          painterResource(R.drawable.ic_opensrplogo),
```
Use the drawable directly. And a human-readable name for test tag. Conventionally we declare a top-level String constant e.g.

```kotlin
const val OPENSR_LOGO_TEST_TAG  =  "opensrpLogoTestTag"
```